Author: Daniel Campello <campello@chromium.org>

Do not resolve -l library paths which causes problems when
cross-compiling.

This fixes issues where something like -lcap was changed to something
like /build/whirlwind/usr/lib/libcap.so and when cross-compiling
architectures did not match and error like this was presented:

/usr/x86_64-pc-linux-gnu/armv7a-cros-linux-gnueabihf/ \
binutils-bin/2.27.0/ld.gold.real: error: /lib/libcap.so.2: incompatible target

 diff --git a/mesonbuild/dependencies/base.py b/mesonbuild/dependencies/base.py
index cd02939..66a7e78 100644
--- a/mesonbuild/dependencies/base.py
+++ b/mesonbuild/dependencies/base.py
@@ -671,9 +671,9 @@ class PkgConfigDependency(ExternalDependency):
                 # Don't resolve the same -lfoo argument again
                 if lib in libs_found:
                     continue
-                if self.clib_compiler:
-                    args = self.clib_compiler.find_library(lib[2:], self.env,
-                                                           libpaths, libtype)
+                #if self.clib_compiler:
+                #    args = self.clib_compiler.find_library(lib[2:], self.env,
+                #                                           libpaths, libtype)
                 # If the project only uses a non-clib language such as D, Rust,
                 # C#, Python, etc, all we can do is limp along by adding the
                 # arguments as-is and then adding the libpaths at the end.
